action.yml

```diff
@@ -19,9 +23,12 @@ runs:
   steps:
     - name: Setup GPR
       shell: bash
-      env:
-        GEM_HOST_API_KEY: "Bearer ${{ inputs.token }}"
       run: |
+        if [ -n "${{ inputs.user}}" ]; then
```

Comment:
won't it always have a value?

Reply:
Nope, I actually need to unset it.
https://github.com/fac/hermod/blob/devp/jenkins2actions/.github/workflows/freeagent-gem.yml#L34

Reply:
```yaml
    - uses: fac/ruby-gem-setup-credentials-action@devp/add-user-input
      with:
        user: ""
        key: rubygems
        token: ${{ secrets.FAC_RUBYGEMS_KEY }}
```

Reply:
The Bearer default is kinda awkward but maintains the v2 interface, otherwise it it will need a v3 (and I want to go on holiday!)